pybind/wallet3/wallet.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../common.hpp"
+
+#include <wallet3/wallet.hpp>
+#include <wallet3/default_daemon_comms.hpp>
+#include <wallet3/keyring.hpp>
+#include <wallet3/config/config.hpp>
+#include <oxenmq/oxenmq.h>
+
+namespace wallet3
+{
+  void
+  Wallet_Init(py::module& mod)
+  {
+    py::class_<wallet::Wallet, std::shared_ptr<wallet::Wallet>>(mod, "Wallet")
+      .def(py::init([](std::string& wallet_name, wallet::Keyring& kr, wallet::Config& config) {
+        auto keyring = std::make_shared<wallet::Keyring>(std::move(kr));
+        auto& comms_config = config.daemon;
+        auto& omq_rpc_config = config.omq_rpc;
+        auto oxenmq = std::make_shared<oxenmq::OxenMQ>();
+        auto comms = std::make_shared<wallet::DefaultDaemonComms>(oxenmq, comms_config);
+        return wallet::Wallet::create(oxenmq, std::move(keyring), nullptr, comms, wallet_name + ".sqlite", "", std::move(config));
```

Comment:
It would make sense to `std::move(...)` the oxenmq and comms shared pointers as well (to make keyring).